packages/datalake-data-export/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "datalake-data-export",
+  "version": "1.0.0",
+  "description": "Data Lake Data Export",
```

Standard in other services
```suggestion
  "description": "PagoPA Interoperability Data Lake Data Export",
```

---

Solved with [544ae08](https://github.com/pagopa/interop-be-monorepo/pull/973/commits/544ae08836685981e6cc7d8ac50bf164c4ac64c8)